src/components/Status/Status.js

```diff
@@ -1,8 +1,14 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
-import { Icon, OverlayTrigger, Button } from 'patternfly-react';
+import { Icon, OverlayTrigger, Button, Popover } from 'patternfly-react';
```

Comment:
please use Popover from '@patternfly/react-core'

it should be easier to use and there should not be a need for `OverlayTrigger` if the trigger is just `onClick`

Reply:
please look at the documentation on how to use it properly: http://patternfly-react.surge.sh/patternfly-4/components/popover/

...I am just going along with the documentation and have not used it myself >.<

Reply:
Using the PF4 Popover component with its built-in header, body, and footer would be great! Here’s another link: https://www.patternfly.org/v4/documentation/react/components/popover/